src/tools/compiletest/src/main.rs

```diff
@@ -479,6 +482,8 @@ fn common_inputs_stamp(config: &Config) -> Stamp {
         stamp.add_path(&rustdoc_path);
         stamp.add_path(&rust_src_dir.join("src/etc/htmldocck.py"));
     }
+    // FIXME(richkadel): Do I need to add an `if let Some(rust_demangler_path) contribution to the
+    // stamp here as well?
```

Comment:
I think you might want to add your python script in here somewhere. Compiled dependencies seem to be handled spearately.

Reply:
Thanks for flagging this. I believe Stamp::add_dir() computes a stamp from all "plain files" in the directory, which should include the python script. I'm not compiling the script; just interpreting it. But just to confirm this, I will do a couple of test runs with and without modifying the script.